include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -66,26 +112,550 @@ class HighPressureGasTransport : public MultiTransport
+    double sigma_mix = 0;
+    double epsilon_over_k_mix = 0;
+    double MW_mix = 0;
+
+    // Values associated with the calculation of the Fc factor in the Chung viscosity model.
+    double mu_mix = 0;
+    double mu_r_mix = 0;
+    double acentric_factor_mix = 0;
+    double kappa_mix = 0;
+};
+
+//! Class ChungHighPressureTransport implements transport properties for
+//! high pressure gas mixtures.
+/*!
+ * The implementation employs a method of corresponding states, using the Takahashi
+ * @cite takahashi1975 approach for binary diffusion coefficients (using mixture
+ * averaging rules for the mixture properties), and the Chung method for the viscosity
+ * and thermal conductivity of a high-pressure gas mixture. All methods are described
+ * in Poling et al. @cite poling2001 (viscosity in Ch. 9, thermal conductivity in
+ * Ch. 10, and diffusion coefficients in Ch. 11).
+ *
+ * @note: All equations that are cited in this implementation are from the 5th edition
+ * of the book "The Properties of Gases and Liquids" by Poling, Prausnitz, and O'Connell.
+ *
+ * @ingroup tranprops
+ */
+class ChungHighPressureGasTransport : public MixTransport
+{
+protected:
+    //! default constructor
+    ChungHighPressureGasTransport() = default;
+
+public:
+    string transportModel() const override {
+        return "high-pressure-chung";
+    }
 
     double viscosity() override;
 
+    /**
+    * Calculates the high-pressure mixture thermal conductivity using the Chung method.
+    *
+    * The Chung method is described in on page 10.23.
+    *
+    * @f[
+    *    \lambda = \frac{31.2 \eta^0 \Psi}{M'} \left( G_1^{-1} + B_6 y \right) + q B_7 y^2 T_r^{1/2} G_2
+    * @f]
+    ** where:
+    *
+    * @f[
+    *   \begin{align*}
+    *       \lambda &= \text{thermal conductivity, W/(m·K)} \\
+    *       \eta^0 &= \text{low-pressure gas viscosity, N·s/m}^2 \\
+    *       M' &= \text{molecular weight, kg/mol} \\
+    *       \Psi &= f(C_v, \omega, T_r) \text{ [as defined under Eq. (10-3.14)]} \\
+    *       q &= 3.586 \times 10^{-3} \left( \frac{T_c}{M'} \right)^{1/2} V_c^{2/3} \\
+    *       T &= \text{temperature, K} \\
+    *       T_c &= \text{critical temperature, K} \\
+    *       T_r &= \text{reduced temperature, } \frac{T}{T_c} \\
+    *       V_c &= \text{critical volume, cm}^3/\text{mol} \\
+    *       \gamma &= \frac{V_c}{6V}
+    *   \end{align*}
+    * @f]
+    *
+    * The details about the parameters and constants used in the expression are
+    * found in the Poling book.
+    *
+    * The mixture values of the pseudo-critical temperature and other model parameters
+    * are calculated using the Chung mixing rules defined on page 9.25.
+    *
+    * The mixture value of the specific heat is computed using equation 10-6.6, which
+    * is the mole fraction weighted sum of the pure species specific heats.
+    *
+    * @f[
+    *   C_{v,m} = \sum_i y_i C_{v,i}
+    * #f]
```

Comment:
This _really_ throws Doxygen off.
```suggestion
     * @f]
     *
```